src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img
+        class="product-card__img"
+        src="./images/imac.jpeg"
+        alt="iMac photo"
+      >
+
+      <h2 class="product-card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="product-card__code">
+        Product code: 195434
+      </p>
+
+      <div class="product-card__rating">
+        <div class="stars stars_star--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="product-card__reviews">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="product-card__pricing">
+        <h3 class="product-card__price-title">
+          Price:
+        </h3>
+        <p class="product-card__price">
+          $2,199
+        </p>
+      </div>
+
+      <a
+        class="product-card__buy-link"
+        href="#buy"
+        data-qa="hover"
+      >
+        <div class="product-card__button">
+          Buy
+        </div>
+      </a>
```

Comment:
Maybe div better to be parent of link tag?
And Buy text should be inside of tag a.

Reply:
Maybe. And I tried, but it broke the active area of the button